### PR TITLE
Fix a broken url in guides

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary

Easy reading and easy navigation

### CODE REVIEW

**Comment by mit-mit** [comment]
Good catch @saintmalik !

**Comment by saintmalik** [comment]
> Good catch @saintmalik !

Thanks 😊

File: src/_guides/testing.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Use the following resources to learn more about testing Flutter apps:
 * [flutter_driver]({{site.flutter_api}}/flutter/flutter_driver/flutter_driver-library.html)<br>
   A testing library for testing Flutter applications on real devices and
   emulators (in a separate process).
-* [flutter/examples/flutter_gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)<br>
+* [flutter_gallery](https://github.com/flutter/gallery)<br>
   Tests for the Flutter gallery example.
```
Comment (kwalrath) [inline src/_guides/testing.md:94]:
```suggestion
  Source code and tests for the Flutter gallery example.
```

**Review by kwalrath — APPROVED** [review]
LGTM, but perhaps we should change the description since the link isn't only to tests?

**Comment by kwalrath** [comment]
Thank you, @saintmalik!

**Pull request merged**